waku/v2/protocol/rln/group_manager/dynamic/web3.go

```diff
@@ -108,78 +128,43 @@ func (gm *DynamicGroupManager) watchNewEvents(ctx context.Context, rlnContract *
 }
 
 const maxBatchSize = uint64(5000)
-const additiveFactorMultiplier = 0.10
-const multiplicativeDecreaseDivisor = 2
 
 func tooMuchDataRequestedError(err error) bool {
 	// this error is only infura specific (other providers might have different error messages)
 	return err.Error() == "query returned more than 10000 results"
```

Comment:
Shouldn't this be an error defined in eth JSON-RPC or am i missing something? 
As of now, it looks like we cannot handle errors from other providers or even if we run a local node.

Reply:
this error is infura specific, and it's not standardized. Looking into geth I could not find the error generated when too many records are returned.